### ISSUE TITLE
Reorder extent of digitization information on IIIF More Information pane

### ISSUE THREAD

**Issue by alisonclemens** [yalelibrary/YUL-DC#2455]
**Story**

As a user, I would like to more clearly understand how much of an object has been digitized when the IIIF viewer is present outside of DCS (e.g., in Archives at Yale, as you can see an example of here: https://archives.yale.edu/repositories/11/archival_objects/334484 -- this item is only partially digitized, but you cannot readily see that in AAY)

Additionally, as a user I see that Extent of Digitization has a value in the object's metadata on the show page but it is not listed in the more information panel, on the generated PDF cover page, or in that object's IIIF manifest.  https://collections.library.yale.edu/catalog/10606932

**Acceptance**
- [x] Extent of digitization information is moved in the IIIF More Information pane, under "Container / Volume Information"
- [ ] Objects like 10606932 are regenerated (perform 'Update IIIF Manifests' for each admin set because we do not know how widespread the inconsistancy is) and display Extent of Digitization in the More Information panel on DCS, on the generated PDF cover page, and embedded on AAY

**Comment by K8Sewell** [comment]
Reorder fields PR - https://github.com/yalelibrary/yul-dc-management/pull/1185

**Comment by K8Sewell** [comment]
Deployed to Test with release [v2.60.7](https://github.com/yalelibrary/yul-dc-management/releases/tag/v2.60.7)

**Comment by K8Sewell** [comment]
Deployed to Test but did not see a change.  Will try deploying blacklight in addition to hopefully pick up the configuration change.

**Comment by K8Sewell** [comment]
The `15475232` record now has extent on the more information panel but it is reporting the container info instead of the extent.  Curiously, this object doesn't have extent info on the show page (should be in description section).  Taking this back to in progress.



![Image](https://user-images.githubusercontent.com/36549923/236537105-df0950ea-8b79-4bb6-9492-88d0386bd8ed.png)


![Image](https://user-images.githubusercontent.com/36549923/236537469-4cb09594-c43f-4125-b873-b6d2c45ecd1d.png)

**Comment by sshetenhelm** [comment]
For 15475232 in UAT, extent of digitization information was not set, which is why it's not showing up.

**Comment by K8Sewell** [comment]
So the question now is how do we have records without the extent of digitization not set, correct?

**Comment by alisonclemens** [comment]
@K8Sewell, @sshetenhelm and I just discussed, and we think we can resolve this data issue for the smaller repositories by regenerating, but regenerating for BRBL will be challenging. Once we can confirm the field got moved when the data is present (for the smaller/non-BRBL repositories), I think we can close this ticket. Early next week I'll write a ticket to investigate how to address the BRBL field appearance failure.

Summer confirms that it's not unexpected to have records without the extent of digitization element, so I don't think we need to worry about that element being lacking in the data
